src/main/java/rs/hybridit/model/Book.java

```diff
@@ -0,0 +1,116 @@
+package rs.hybridit.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+import javax.persistence.*;
+import javax.persistence.Entity;
+import javax.persistence.Table;
+import java.util.HashSet;
+import java.util.Set;
+
+@Entity
+@Table(catalog = "dbhybridlibrary", name = "books")
+public class Book {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(name = "isbn")
+    private Integer isbn;
+
+    @Column(name = "name")
+    private String name;
+
+    @Column(name = "author")
+    private String author;
+
+    @Column(name = "image")
+    private String image;
+
+    @Column(name = "renting_counter")
+    private Integer rentingCounter;
+
+    @JsonIgnore
+    @OneToMany(mappedBy = "book", fetch = FetchType.LAZY, cascade = CascadeType.REFRESH)
```

Comment:
this should be CascadeType.ALL

https://stackoverflow.com/questions/17087164/what-is-cascading-in-hibernate